setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 format = pylint
-exclude = .svc,CVS,.bzr,.hg,.git,__pycache__,venv,databuilder/sql_parser/usage/presto/antlr_generated
+exclude = .svc,CVS,.bzr,.hg,.git,__pycache__,venv,build,databuilder/sql_parser/usage/presto/antlr_generated
```

Comment:
The `build` directory essentially contains a copy of our code. The `databuilder/sql_parser...` rule only matched the original copy but did not match the copy in `build/lib/databuilder/sql_parser...`. I think it's safe to exclude the `build` from the linter.